arrow_test.go

```diff
@@ -309,6 +310,94 @@ func TestArrowFloatPrecision(t *testing.T) {
 	})
 }
 
+func TestArrowTimePrecision(t *testing.T) {
+	runTests(t, dsn, func(dbt *DBTest) {
+		dbt.mustExec("CREATE TABLE t (col5 TIME(5), col6 TIME(6), col7 TIME(7), col8 TIME(8));")
+		defer dbt.mustExec("DROP TABLE IF EXISTS t")
+		dbt.mustExec("INSERT INTO t VALUES ('23:59:59.99999', '23:59:59.999999', '23:59:59.9999999', '23:59:59.99999999');")
+
+		ctx := WithHigherPrecision(context.Background())
```

Comment:
I might miss something, the changes in converter.go seems not under higher precision?

Reply:
Good catch - we actually don't need higher precision for this, just realizing.